static/app/actionCreators/drawer.tsx

```diff
@@ -0,0 +1,13 @@
+import type {DrawerOptions, DrawerRenderProps} from 'sentry/components/globalDrawer';
+import DrawerStore from 'sentry/stores/drawerStore';
+
+export function openDrawer(
+  renderer: (renderProps: DrawerRenderProps) => React.ReactNode,
+  options?: DrawerOptions
+) {
+  DrawerStore.openDrawer(renderer, options ?? {});
+}
+
+export function closeDrawer() {
+  DrawerStore.closeDrawer();
+}
```

Comment:
i wouldn't add any new action creators, its the same thing as using the DrawerStore directly